frontend/app/src/components/display/BooleanDisplay.vue

```diff
@@ -0,0 +1,18 @@
+<script setup lang="ts">
+withDefaults(
+  defineProps<{
+    display: boolean;
```

Comment:
maybe we could rename this to `success`? 

I would expect display false to mean hidden.

Maybe also rename it to `SuccessDisplay` or something?